featuretools/entityset/serialize.py

```diff
@@ -140,14 +141,28 @@ def write_data_description(entityset, path, profile_name=None, **kwargs):
         dump_data_description(entityset, path, **kwargs)
 
 
+class NpJsonEncoder(json.JSONEncoder):
```

Comment:
It looks like the issue is that `interesting_values` is a list with numpy values in it.  What if instead of creating this custom encoder, we force `interesting_values` to be a pandas series?

Users can set interesting_values manually so we'd need to coerce to a series if they passed a different type.